bibcat/llm/metrics.py

```diff
@@ -118,6 +114,22 @@ def extract_eval_data(data: dict, missions: list[str]) -> dict[str, Any]:
                 llm_df_missions,
                 n_human_llm_hallucination,
             )
+            continue
```

Comment:
Is this meant to be here?  You don't need `continue` statements at the end of if-else blocks, unless you  want it to skip running any other code within the for loop.